mySC.js

```diff
@@ -1,13 +1,22 @@
 $(document).ready(function() {
 
   // Queue up a SoundCloud song to play via the song url
-  var song = 'https://soundcloud.com/chancetherapper/acid-rain-1';
-  playSC(song, false); // false = don't autoPlay, true = autoPlay
-  getSCinfo(song);
+  var song1 = 'https://soundcloud.com/chancetherapper/acid-rain-1';
+  var song2 = 'https://soundcloud.com/hxns/precious';
+  var song3 = 'https://soundcloud.com/morguemami/limelight-prod-xauve';
+  var song4 = 'https://soundcloud.com/joshpanii/usedii';
+
+  playSC(song1, false); // false = don't autoPlay, true = autoPlay
+
+  getSCinfo(song1, 'thumbnail', true, true);
+  getSCinfo(song2, 'smallThumb1');
```

Comment:
Loop through these lines where the stop conditional is the number of items in our song array.